### ISSUE TITLE
Failed to import collection from link

### ISSUE THREAD

**Issue by benschmi** [a85/POSTMan-Chrome-Extension#7114]
**Describe the bug**
Importing a swagger file from link fails with the error message "Error while importing Swagger 2.0: Could not import" in Postman 7.5.0 for Mac (works for Windows and Linux)

**To Reproduce**
1. Click on 'Import'
2. Click on 'Import from Link'
3. Add "http://api.ionos.com/cloudapi/v5/swagger.json" as destination
4. Click on "Import" 
5. See error

**Expected behavior**
Collection is imported.

**App information (please complete the following information):**
 - App Type: Native MacOs App
 - Postman Version 7.5.0
 - OS:  macOS Mojave 10.14.5

**Comment by nlozanoarguelles** [comment]
I am experiencing the same problem importing [this Swagger 2.0](https://raw.githubusercontent.com/AdobeAtAdobe/kirby_docs/master/acpdr/swagger-specs/decisioning-repository.yaml) definition.

**App information:**
App Type: Native MacOs App
Postman Version 7.5.0
OS: macOS Mojave 10.14.5

**Comment by jaydorsey** [comment]
I imported a swagger file about an hour ago, then I updated postman and that same swagger file generates the same error. 

I was quite a few versions of postman behind tbh so I don't know what version I _was_ on, but it's 7.7.0 now. Mojave 10.14.6

The logs don't provide any clue as to the problem:

```
[92669][1568393006499][requester][error]["Error importing Swagger 2.0 spec"]
[92669][1568393006500][requester][error]["Swagger 2 ~ Failed to import Swagger 2.0","Error importing Swagger 2.0 spec"]
[92669][1568393270471][requester][error]["Error importing Swagger 2.0 spec"]
[92669][1568393270471][requester][error]["Swagger 2 ~ Failed to import Swagger 2.0","Error importing Swagger 2.0 spec"]
```

Edit: I used the [swagger editor](https://editor.swagger.io) to lint my swagger definition and I had a couple structural warnings. Once I fixed those, I was able to import my API endpoint. I'm guessing a validation step was added to postman recently.

The error message when importing swagger definitions could be more useful. I'm trying to bridge the gap between our current process, generating swagger docs automatically, and using postman and I can't recreate everything we've already automated from scratch due to a minor linting issue (it was an extra property). It was pretty easy to troubleshoot with the swagger tools
